### PR TITLE
implemented selection attachments

### PR DESCRIPTION
**Pull request by mxrcury**
implementation of the selection:

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/90138904/f468c1ea-17f2-4707-ac58-7580aa4f0244

### CODE REVIEW

File: src/components/enhanced-table/enhanced-table-row/EnhancedTableRow.jsx [hunk 1]
```diff
@@ -51,13 +53,21 @@ const EnhancedTableRow = ({
...
   return (
     <TableRow
       hover
       key={item._id}
       onClick={handleRowClick}
       selected={isSelection && isSelected(item._id)}
-      sx={{ cursor: onRowClick && 'pointer' }}
+      sx={{
+        cursor: onRowClick && 'pointer',
+        ...(isRowSelected && styles.selectedRow)
```
Comment (Tolik170) [inline src/components/enhanced-table/enhanced-table-row/EnhancedTableRow.jsx:69]: move to styles

**Review by Tolik170 — APPROVED** [review]

File: src/components/enhanced-table/enhanced-table-row/EnhancedTableRow.jsx (hunk 1 above)
Comment (mxrcury) [reply src/components/enhanced-table/enhanced-table-row/EnhancedTableRow.jsx]: moved

File: src/containers/add-attachments/AddAttachments.tsx [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+  onAddAttachments: (attachments: Attachment[]) => void
+}
+
+const AddAttachments: FC<AddAttachmentsProps> = ({
+  attachments,
+  onAddAttachments
+}) => {
   const [inputValue, setInputValue] = useState<string>('')
+  const [selectedAttachments, setSelectedAttachments] = useState<Attachment[]>(
+    attachments.length ? attachments : []
```
Comment (Tolik170) [inline src/containers/add-attachments/AddAttachments.tsx:37]: do we need this check, maybe attachments will be an empty array as a default response?

Comment (mxrcury) [reply src/containers/add-attachments/AddAttachments.tsx]: yeap, it would looks good so i have added default value

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**